include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -218,8 +218,31 @@ class CommonOperatorLoader : public ProtobufLoader {
     } else {
       Tensor *T = getTensorByName(op.input(1));
       auto TH = T->getHandle<size_t>();
+
+      std::vector<int64_t> protoDims(T->size());
+      auto oldDim = in->dims(0);
```

Comment:
`in->dims(0)` gets the `dims` of the first output of `in`. Instead, use `in.dims()` here, which will use whatever output of `in` is being referred to here, since `in` is a `NodeValue`.